Firestore/core/src/firebase/firestore/immutable/sorted_set.h

```diff
@@ -74,6 +74,22 @@ class SortedSet : public SortedContainer {
     return SortedSet{map_.insert(key, {})};
   }
 
+  ABSL_MUST_USE_RESULT SortedSet union_with(const SortedSet<K>& other) const {
```

Comment:
I think you can drop the `<K>` here. You get it for free by virtue of being in the declaration of `SortedSet<K>`.

Reply:
Done.